scripts/build/rollup-plugins/executable.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
```

Comment:
https://github.com/developit/rollup-plugin-preserve-shebang?

Reply:
Looks like it does the same thing, minus *chmod*ing the file (+x)